tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/HealthCheckSteps.cs

```diff
@@ -22,8 +22,8 @@ public HealthChecksSteps(Response response, ScenarioContext context, Settings se
             _context["organisationBaseUrl"] = _settings.OrganisationApiBaseUrl;
         }
 
-        [Given(@"The Smtp Server is (up|down)")]
-        public void GivenTheSmptSeverIsInState(string state)
+        [Given(@"The (Smtp|Database) Server is (up|down)")]
+        public void GivenTheServerIsInState(string server, string state)
```

Comment:
The `server` parameter isn't currently used – should it be?

Reply:
The param is there only to satisfy specflow. The step would be functional without it - being just 'The Server is up/down', however I thought adding 'Smtp/Database' would give a bit more context.

There's a few ways to go about this, however I'm not sure on which one is the best.
1) Have 2 separate steps - The Smtp Server is up/down & The Database Server is up/down. These two steps would do exactly the same thing as the current method.
2) Have 1 step 'The Server is up/down' and would do what the current method does.
3) The current way with a redundant parameter.

1) Feels like violating DRY
2) Feels bit too vague
3) is not the best but better than 1 & 2

What do you think?

Reply:
The given template is just a regex isn't it so can you just remove the capture group, so the attribute becomes `[Given(@"The Smtp|Database Server is (up|down)")]`?

Reply:
That's fantastic! I didn't know about that feature, thank you Chris, I'll change it to that.

Reply:
My mistake with the suggestion here. I should have suggested a non-capturing group:

```c#
[Given(@"The (?:Smtp|Database) Server is (up|down)")]
```